pkg/bbgo/order_executor_general.go

```diff
@@ -144,6 +144,20 @@ func (e *GeneralOrderExecutor) GracefulCancelActiveOrderBook(ctx context.Context
 	return nil
 }
 
+func (e *GeneralOrderExecutor) Cancel(ctx context.Context, order types.Order) error {
```

Comment:
call it `CancelOrder`?

Reply:
call it `GracefulCancelOrder` instead.  
we already have `CancelOrders` which directly cancels the orders from the Exchange.